targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/TARGET_M33_NS/device/TOOLCHAIN_GCC_ARM/LPC55S69_cm33_core0_flash.ld

```diff
@@ -57,15 +57,19 @@ __ram_vector_table__ = 1;
     #define MBED_BOOT_STACK_SIZE 0x400
 #endif
 
+#if !defined(MBED_NVIC_SIZE)
```

Comment:
Why is the define MBED_NVIC_SIZE used only for the GCC linker script?

Reply:
Because it already uses the macro `m_interrupts_size`.